config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace-core/1_event_attributes.conf

```diff
@@ -1,4 +1,4 @@
-version = 2
+version = 1
```

Comment:
Hypertrace Core attributes should always have version as 1

Reply:
I don't think we should have that requirement - updating versions is sometimes required. We should be naming different sets of attributes differently so we don't have any version conflicts. If we need ordering between those sets, we can adjust the config to order them - but as a last resort.

Reply:
When do we update a version?

Reply:
A version update is required to trigger an update if the attributes haven't changed (which sometimes happens if sequencing gets out of order, a change in the client or server updates the semantics of existing config, etc.)

Reply:
I see. I was wondering whether should I revert the versions back to the older versions. But, downgrading the versions also seems fine

I can have unique names then and prefix the names with `htc` and `ht`, so that we have unique names

Reply:
Yeah, no need to revert but we shouldn't rely on versions sticking being at any particular value.